code/modules/cargo/console.dm

```diff
@@ -168,6 +168,8 @@
 				log_game("[key_name(usr)] accepted a shuttle loan event.")
 				. = TRUE
 		if("add")
+			if(express) //prevents running this in the wrong way
+				return . = FALSE
```

Comment:
You want either `return FALSE` or `. = FALSE`, depending on if you want it to return here or not.